ee/fleetd-chrome/src/tables/screen_lock.ts

```diff
@@ -0,0 +1,20 @@
+import Table from "./Table";
+export default class TableScreenLock extends Table {
+  name = "screen_lock";
+  columns = ["delay"];
```

Comment:
osquery's [screenlock](https://osquery.io/schema/5.8.2/#screenlock) has two columns, `enabled` and `grace_period`.

Should we add a enabled column for Chrome OS?

On my Chromebook I can see a toggle to disable lock screen.

Also, any reason it's `screen_lock` vs `screenlock`?

Reply:
Lucas
  [16 minutes ago](https://fleetdm.slack.com/archives/C01EZVBHFHU/p1686860416291599?thread_ts=1686859659.666889&cid=C01EZVBHFHU)
Ah, there's no bug because the table doesn't work when osquery runs as root, which is the case for Fleet deployments. But the table works when using osqueryi as your user (not root)